### PR TITLE
add options to babel-preset-latest

### PR DESCRIPTION
**Pull request by hzoo**
- [ ] tests

### CODE REVIEW

File: packages/babel-preset-latest/package.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
   "dependencies": {
```
Comment (danez) [inline packages/babel-preset-latest/package.json]: the main is now pointing to the wrong file

File: packages/babel-preset-latest/src/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = function preset(context, opts = {}) {
+  return {
+    presets: [
+      opts.es2015 !== false && [require("babel-preset-es2015"), opts.es2015],
```
Comment (danez) [inline packages/babel-preset-latest/src/index.js]: So one could disable es2015 now by setting es2015: false? is that desired?

Comment (hzoo) [inline packages/babel-preset-latest/src/index.js]: yeah an idea what do you think

File: packages/babel-preset-latest/package.json (hunk 1 above)
Comment (hzoo) [inline packages/babel-preset-latest/package.json]: cool 👍 , and should be caught with the tests 😄  I hope

File: packages/babel-preset-latest/src/index.js (hunk 2 above)
Comment (danez) [inline packages/babel-preset-latest/src/index.js]: yeah thats a nice idea, then it should be also easy for people to use this preset env aware kindof by only enabling 2016/2017 for node 6 e.g.

**Comment by hzoo** [comment]
Adding tests in the next PR to make master pass

**Pull request merged**